src/component/button.rs

```diff
@@ -1,7 +1,7 @@
 use crate::util::Color;
 use yew::prelude::*;
 
-#[derive(Clone, PartialEq)]
+#[derive(Clone, PartialEq, Eq)]
```

Comment:
Just obeying `clippy` to make the suggestion go away. Might cause problems if we make changes to the enum that make `Eq` no longer derivable.